lib/bolt/transport/local/connection.rb

```diff
@@ -47,6 +47,7 @@ def copy_file(source, dest)
 
         def execute(command)
           if Bolt::Util.windows?
+            command = Bolt::Shell::Powershell::Snippets.exit_with_code(command)
```

Comment:
This should be in the conditional below, so we only append the exit code stuff if we're goign to wrap it in 
`powershell.exe -Command`. Otherwise, it should already be a properly-formatted `powershell.exe` invocation.